src/components/App/App.module.css

```diff
@@ -1,21 +1,85 @@
+body {
+  background-image: url('./Media/wall4.jpg');
+  min-height: 100vh;
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 100%;
+}
+
 .wrap {
-  display: grid;
+  display: flex;
+  flex-flow: column;
   margin: auto;
-  width: 600px;
+  max-height: 90%;
   padding: 50px;
-  grid-template-areas: "sidebar ... content content";
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  font-family: 'Poppins', sans-serif;
+  font-size: 1 rem;
+  background-color: white;
+  width: 100%;
+  padding-left: 10px; 
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/ReactProject_karolina/pullRequest?prid=6313319)